mysql-test/t/ssl_bug75311.test

```diff
@@ -14,7 +14,7 @@ SHOW STATUS LIKE 'Ssl_cipher_list';
 --connection default
 
 # The first error string is returned by YaSSL, the second one by OpenSSL
```

Comment:
Three replacements now but the comment explains only the two of them (this comment applies to lower versions too)